application/etc/database/changelog.yml

```
@@ -21,3 +21,14 @@ databaseChangeLog:
             splitStatements: true
+  - changeSet:
+      id: 3
+      author: darshilshah2017
+      comment: "Create partitioned table 'stock_analysis.daily_index_data' and initialize pg_partman partitioning with 3-month range partitions"
+      changes:
+        - tagDatabase:
+            tag: SAS-7
+        - sqlFile:
+            path: SAS-7.sql
+            relativeToChangelogFile: true
+            splitStatements: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)